react/MuiCozyTheme/overrides/makeLightNormalOverrides.js

```diff
@@ -455,7 +455,7 @@ export const makeLightNormalOverrides = theme => ({
           maxWidth: '800px'
         }
       },
-      '&.overflow': {
+      '&.overflow, &.overflow .cozyDialogContent': {
```

Comment:
si ça t'ennuie pas je préfère séparer les éléments, c'est ce qu'on fait partout et je trouve que la lisibilité est meilleur, ça évite de rater un truc en lisant vite...

```
'&.overflow': {
  overflowY: 'visible !important' // Allow the icon to overflow the dialog, otherwise it will be cut off,
  '& .cozyDialogContent': {
    overflowY: 'visible !important'  // Allow the icon to overflow the dialog, otherwise it will be cut off
  }
}
```

Par contre ça pose quand même question 🤔 est-ce que les deux sont vraiment nécessaires ?

Reply:
Je check ça

Reply:
Unfortunately I need both conditions for it to work. I separated it as you suggest